test/e2e/authorizer/authorizer_test.go

```diff
@@ -190,6 +189,37 @@ func TestAuthorizer(t *testing.T) {
 				return true
 			}, wait.ForeverTestTimeout, time.Millisecond*100, "User-3 should now be able to list Namespaces")
 		},
+		"without org access, a deep SAR with user-1 against org2 succeeds even without org access for user-1": func(t *testing.T) {
+			t.Logf("try to list ConfigMap as user-1 in %q without access, should fail", org2.Join("workspace1"))
+			_, err := user1KubeClusterClient.CoreV1().ConfigMaps("default").List(logicalcluster.WithCluster(ctx, org2.Join("workspace1")), metav1.ListOptions{})
+			require.Errorf(t, err, "user-1 should not be able to list configmaps in %q", org2.Join("workspace1"))
```

Comment:
(no action required) can use `Error` instead of `Errorf` because they're identical (same for `Equalf`, `Falsef`, etc)